DataFormats/Candidate/src/classes_def.xml

```diff
@@ -55,12 +55,36 @@
     <version ClassVersion="10" checksum="2404784677"/>
    </class>
 
-  <class name="reco::VertexCompositePtrCandidate" ClassVersion="13">
+  <class name="reco::VertexCompositePtrCandidate" ClassVersion="14">
+   <version ClassVersion="14" checksum="2052765294"/>
    <version ClassVersion="13" checksum="3584734005"/>
    <version ClassVersion="12" checksum="3253990857"/>
    <version ClassVersion="11" checksum="2070976353"/>
    <version ClassVersion="10" checksum="955269040"/>
   </class>
+  <ioread sourceClass="reco::VertexCompositePtrCandidate" version="[-13]" targetClass="reco::VertexCompositePtrCandidate" 
+	  source="Double32_t chi2_" 
+          target="chi2_">    
+    <![CDATA[chi2_ = onfile.chi2_;]]>
+  </ioread>
+  <ioread sourceClass="reco::VertexCompositePtrCandidate" version="[-13]" targetClass="reco::VertexCompositePtrCandidate" 
+	  source="Double32_t ndof_" 
+          target="ndof_">  
+    <![CDATA[ndof_ = onfile.ndof_;]]>
+  </ioread>
+  <ioread sourceClass="reco::VertexCompositePtrCandidate" version="[-13]" targetClass="reco::VertexCompositePtrCandidate" 
+	  source="Double32_t covariance_[6]" 
+          target="covariance_">
+    <![CDATA[
+	   for( unsigned i = 0; i < 4; ++i ) { for( unsigned j = i; j< 4; ++j ) { unsigned a = (i <= j ? i : j), b = (i <= j ? j : i); unsigned idx = b * (b + 1)/2 + a; covariance_[idx] = ( i == 3 || j == 3 ) ? 0.0 : onfile.covariance_[idx]; } }
+    ]]>
+  </ioread>
+  <ioread sourceClass="reco::VertexCompositePtrCandidate" version="[-13]" targetClass="reco::VertexCompositePtrCandidate" 
+	  source="" 
+          target="time_">
+    <![CDATA[time_ = 0.;]]>
```

Comment:
is this really needed?
It looks like a more complex replacement to missing initialization in the default constructor.
Please try to add a default value in the VertexCompositePtrCandidate.h file instead